.env

```diff
@@ -27,6 +20,15 @@ APPLICATION_CONFIG_PATH=./compose-network/network-node/data/config
 #### Network Node Memory Limits ####
 NETWORK_NODE_MEM_LIMIT=8gb
 
+#### Uploader Image Prefixes & Tags ####
+UPLOADER_IMAGE_PREFIX=gcr.io/hedera-registry/
+UPLOADER_IMAGE_TAG=0.5.0
+
+#### MirrorNode Prefixes & Tags ####
+MIRROR_IMAGE_PREFIX=gcr.io/mirrornode/
+MIRROR_IMAGE_TAG=0.56.3
```

Comment:
Use v0.57.1 as it has most of the relay functionality

Reply:
fixed